pkg/confmap/config.go

```diff
@@ -0,0 +1,9 @@
+package confmap
```

Comment:
@srikanthccv can we find a better place for this interface?

Reply:
Maybe move to `pkg/config/config`?

Reply:
Some circular import issue might come up! Letting it be for now